store/mapSlice.ts

```diff
@@ -0,0 +1,82 @@
+import { createSelector, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Location } from 'types/locations';
+
+import { RootState } from 'store';
+
+type State = {
+  locations: Location[];
+  currentLocationIndex: number;
+};
+
+const initialState: State = {
+  locations: [
+    {
+      name: 'Kyiv, Ukraine',
+      address: { label: 'Kyiv, Ukraine, Mechnykova street,', link: 'https://goo.gl/maps/8YZ1CDQC31ChoSjQ9' },
+      coordinates: {
+        lat: 50.43774939,
+        lng: 30.52727746,
+      },
+      contact: {
+        title: 'Client Manager',
+        fullName: 'Olena Makovets',
+        email: { label: 'hello@zapal.tech', link: 'mailto:ivan.salata@zapal.tech' },
```

Comment:
Change URLs.
Titles: "Country manager"
Kyiv email: front-office@
Warsaw/Quebec maps url: (some business center in Warsaw/Quebec)